README.md

```diff
@@ -85,12 +89,12 @@ Considering the example above this call will return the following JSON:
 Here is a list of common JavaScript libraries and the time it takes to parse them, 
 compared to the time it took for the same script using the original Esprima in Chrome.
 
-| Script | Size | Esprima .NET | Esprima (Chrome) |
-| --- | --- | --- | --- |
-| underscore-1.5.2 | 43 KB | 2.4 ms | 3.1 ms |
-| backbone-1.1.0 | 60 KB | 2.9 ms | 3.5 ms |
-| mootools-1.4.5 | 163 KB | 18.7 ms | 16.2 ms | 
-| jquery-1.9.1 | 271 KB | 22.8 ms | 19.0 ms |
-| yui-3.12.0| 341 KB | 17.2 ms | 16.2 ms |
-| jquery.mobile-1.4.2 | 456 K | 43.3 ms | 46.9 ms | 
-| angular-1.2.5 | 721 KB | 29.3 ms | 37.2 ms |
+| Script              | Size   | Esprima .NET (.NET 6) | Esprima (Chrome 116) |
+| ------------------- | ------ | ----------------------| -------------------- |
+| underscore-1.5.2    | 43 KB  | 1.4 ms                | 1.0 ms               |
```

Comment:
hmm, a bit weird to see that JS version can now go faster... 🤷🏻‍♂️

Reply:
That'd be weird indeed, so I've double-checked the results. Luckily, I just inverted the sides 🤦

Reply:
Whoa that was a close one 🤣